bundle/manifests/backstage-operator.clusterserviceversion.yaml

```diff
@@ -300,6 +300,8 @@ spec:
   maintainers:
   - email: asoro@redhat.com
     name: Armel Soro
+  - email: cdaley@redhat.com
+    name: Corey Daley
```

Comment:
if you're adding Corey and removing Jianrong in the pr build, you could also remove Jianrong here.

Reply:
Good catch. Fixed in [`30b044b` (#364)](https://github.com/janus-idp/operator/pull/364/commits/30b044b07d8b9b74bc58cccc034b28f0ca3c0ce0)